plugins/org-react/src/components/GroupListPicker/GroupListPicker.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback } from 'react';
+import {
+  catalogApiRef,
+  humanizeEntityRef,
+} from '@backstage/plugin-catalog-react';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import useAsync from 'react-use/lib/useAsync';
+import Popover from '@material-ui/core/Popover';
+import { useApi } from '@backstage/core-plugin-api';
+import { ResponseErrorPanel } from '@backstage/core-components';
+import { Entity, GroupEntity } from '@backstage/catalog-model';
+import { GroupListPickerButton } from './GroupListPickerButton';
+
+/**
+ * Props for {@link GroupListPicker}.
+ *
+ * @public
+ */
+export type GroupListPickerProps = {
+  placeholder?: string;
+  groupTypes?: Array<string>;
+  initialGroup?: string | undefined;
```

Comment:
This one is a bit weird to me, it seems like it just sets the label of the button and doesn't actually have an impact on which group is selected? Perhaps it should be dropped and have it be a completely uncontrolled component instead, with it displaying the currently selected team. It also conflicts a bit with the placeholder I think? even though that one's for the dropdown